book/src/drivers.md

```diff
@@ -91,7 +90,9 @@ Name | CSI Driver Name | Compatible with CSI Version(s) | Description | Persiste
 [Sangfor-EDS-Block-Storage](https://github.com/eds-wzc/sangfor-eds-csi) | `eds.csi.block.sangfor.com` | v1.0 | A Container Storage Interface (CSI) Driver for Sangfor Block Storage(EDS) | Persistent | Read/Write Single Pod | Yes |
 [SeaweedFS](https://github.com/seaweedfs/seaweedfs-csi-driver) | `seaweedfs-csi-driver` | v1.0 | A Container Storage Interface (CSI Driver for [SeaweedFS](https://github.com/chrislusf/seaweedfs)) | Persistent | Read/Write Multiple Pods | Yes |
 [Secrets Store CSI Driver](https://github.com/kubernetes-sigs/secrets-store-csi-driver) | `secrets-store.csi.k8s.io` | v0.0.10 | A Container Storage Interface (CSI) Driver for mounting secrets, keys, and certs stored in enterprise-grade external secrets stores as volumes. | Ephemeral | N/A | N/A | 
-[SmartX](http://www.smartx.com/?locale=en) | `csi-smtx-plugin` | v1.0 | A Container Storage Interface (CSI) Driver for SmartX ZBS Storage  | Persistent | Read/Write Multiple Pods | Yes | Snapshot, Expansion
+[SmartX](http://www.smartx.com/?locale=en) | `csi-smtx-plugin` | v1.0 | A Container Storage Interface (CSI) Driver for SmartX ZBS Storage  | Persistent | Read/Write Multiple Pods | Yes | Snapshot, Expansion  
+[SODA](https://github.com/sodafoundation/nbp/tree/master/csi) | `csi-opensdsplugin` | v1.0 | A Container Storage Interface (CSI) Driver for [SODA](https://sodafoundation.io/) | Persistent | Read/Write Single Pod | Yes | Raw Block, Snapshot  
+[SODA-CSI-PnP](https://github.com/sodafoundation/nbp/tree/master/csi-plug-n-play/sidecars) | `soda-csi` | v1.2 | A CSI Plug-N-Play mechanism for all existing CSI drivers to work with  [SODA](https://sodafoundation.io/) Profiles | Persistent | Read/Write Single Pod | Yes | Raw Block, Snapshot  
```

Comment:
Looks like these are modified sidecars based on upstream sidecars.  This is not a CSI Driver so we should not have an independent entry for it here.  Instead you can add a link to it in the SODA CSI Driver doc.

Reply:
@xing-yang , this will have additional metadata management with SODA to manage all the TP Drivers to provide additional data management features and unified management of drivers. Do you think, still we keep it separately out of the CSI drivers listing....?!

Reply:
@xing-yang @skdwriting I have removed the entry of Soda CSI Plug-N-Play, so this PR just updates the name of Opensds plugin to Soda CSI plugin, Plug-N-Play addition can be proposed in another PR after discussing in the community